spec/controllers/works/default_rails_actions_spec.rb

```diff
@@ -325,43 +404,61 @@ def call_with_params(params)
 
       context "with a valid owner tag" do
         before do
-          @fandom2 = create(:canonical_fandom)
-          @work2 = create(:work, fandom_string: @fandom2.name)
+          fandom2 = create(:canonical_fandom)
+          work2 = create(:work, fandom_string: fandom2.name)
           run_all_indexing_jobs
         end
 
         it "only gets works under that tag" do
-          get :index, params: { tag_id: @fandom.name }
-          expect(assigns(:works).items).to include(@work)
-          expect(assigns(:works).items).not_to include(@work2)
+          get :index, params: { tag_id: fandom.name }
+          expect(assigns(:works).items).to include(work)
+          expect(assigns(:works).items).not_to include(work2)
         end
 
         it "shows different results on second page" do
           get :index, params: { tag_id: @fandom.name, page: 2 }
-          expect(assigns(:works).items).not_to include(@work)
+          expect(assigns(:works).items).not_to include(work)
+        end
+
+        context "when disabling filtering" do
```

Comment:
I don't think this accurately describes what's being tested. It looks like this is testing when the `suspend_filter_counts` setting is enabled -- disabling filters was removed in #3546.

Reply:
Fixed